logger/logger.go

```diff
@@ -64,6 +64,10 @@ type logger struct {
 	Logger kitlog.Logger
 }
 
+func (l *logger) Decorate(keyvals ...interface{}) {
+	l.Logger = kitlog.NewContext(l.Logger).With(keyvals...)
```

Comment:
I do not think that this will work out because it effectively changes the whole logger. We use the same logger across parallel requests. How we use the logger currently is not thread safe. This is why we have to add informations per requests to the log call directly. AFAIK you want to do this because the cluster ID in the aws-operator is always the same across a reconciliation. So we have to do this. 

```
s.Log.("cluster-id", clusterID)
```